src/main/java/uk/gov/hmcts/ccd/v2/V2.java

```diff
@@ -12,6 +12,7 @@ private MediaType() {}
         // External API
         public static final String CASE = "application/vnd.uk.gov.hmcts.ccd-data-store-api.case.v2+json;charset=UTF-8";
         public static final String CASE_DOCUMENTS = "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-documents.v2+json;charset=UTF-8";
+        public static final String CASE_DOCUMENT = "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-document+json;charset=UTF-8;version=1.0";
```

Comment:
please let's change this to: `"application/vnd.uk.gov.hmcts.ccd-data-store-api.case-document.v1+json;charset=UTF-8";`